ofl/quicksand/DESCRIPTION.en_us.html

```diff
@@ -3,4 +3,5 @@
 The project was initiated by Andrew Paglinawan in 2008 using geometric shapes as a core foundation. 
 It is designed for display purposes but kept legible enough to use in small sizes as well. 
 In 2016, in collaboration with Andrew, it was thoroughly revised by Thomas Jockin to improve the quality. 
+In 2019 Mirko Velimirovic made Quicksand Variable along the weight axis.
```

Comment:
Would prefer "In 2019, Mirko Velimirovic converted the family into a variable font."

Reply:
Updated. Thanks for the help Marc!